api/v1/routes/auth.py

```diff
@@ -173,10 +183,66 @@ async def verify_signin_token(token_schema: TokenRequest, db: Session = Depends(
             "token_type": "bearer"
         }
     )
-    
+
 
 # Protected route example: test route
 @auth.get("/admin")
 def read_admin_data(current_admin: Annotated[User, Depends(user_service.get_current_super_admin)]):
     return {"message": "Hello, admin!"}
 
+def generate_access_token(data: dict, expires_delta: Optional[timedelta] = None):
```

Comment:
From here, all of this needs to be in user service. 
Generate access token has been taken care of

Reply:
Once it's put there, it does not work again for some reasons I can't explain, so I had to leave the functions in the auth file.